hermes/probe/error/error.go

```diff
@@ -30,22 +30,21 @@ import (
 
 // ProbeError is an error that includes an exit status for used within Hermes.
 type ProbeError struct {
-	// TODO(evanSpendlove): Refactor metrics.ExitStatus to use something similar to metrics.APICallStatus.
+	// TODO(evanSpendlove): Refactor metrics.ExitStatus to be named something similar to metrics.APICallStatus.
 	Status metrics.ExitStatus
 	Err    error
 }
 
-// New returns a new ProbeError containing the error passed
-// with additional context.and the status.
+// New returns a new ProbeError containing the error and status passed.
 // Arguments:
 //	- status: pass the exit status associated with this error.
-//	- innner: pass the inner error to be wrapped.
+//	- err: pass the error to be embedded.
```

Comment:
Nit: "embedding" has a particular meaning in Go. I would jus say "err: the underlying error".

Reply:
Okay, I've changed this in the latest commit.